maltd-frontend/cypress/integration/tests/find-users.js

```diff
@@ -0,0 +1,39 @@
+/* global cy, before */
+// / <reference types="Cypress" />
+
+import LandingPage from "../../support/pageObjects/landing-page";
+
+describe("The landing Page", () => {
+  before(() => {
+    // Runs once before all tests in the block
+    cy.fixture("userData").then(data => {
+      this.data = data;
+    });
+  });
+
+  it("Find users", () => {
+    const landingPage = new LandingPage();
+    // Launches the url
+    cy.visit("/");
+
+    // Validates the Find button is disabled by default
+    landingPage.getFindButton().should("be.disabled");
+
+    // Validating invalid inputs and error message
+    let i = this.data.userName.invalid.length;
```

Comment:
question: can any of these values return `undefined`? like for example if `invalid` is undefined and now we cant do `.length` of that? just curious - if not, then all good!

Reply:
Did not come across ```undefined```return value yet. There is a possibility of code change due to the errors. I will keep this in mind. Thanks.